src/test_report.py

```diff
@@ -27,6 +32,69 @@ def __init__(self, configs, args):
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.db.get_db(self._db_config, api_token)
         self._logger = Logger("config/logger.conf", "test_report")
+        self._email_host = args.email_host
+        self._email_port = int(args.email_port)
+        self._email_send_from = args.email_send_from
+        self._email_subject = args.email_subject
+        self._email_send_to = args.email_send_to
```

Comment:
I don't think this will work. It expects configuration to be in the `kernelci.conf` or passed in the CLI, but the appropriate config is actually in `config/pipeline.yaml`

Reply:
I thought we had already agreed to have the from/to/subject hard-coded in the script anyway and leave that for later to add as configuration?

Reply:
Yes, there have been this agreement, but I can see in this code that these values are actually in the `pipeline.yaml` file. 
Let's just hardcode the values and remove the changes from `pipeline.yaml`